functions/New-DbaAgentOperator.ps1

```diff
@@ -0,0 +1,327 @@
+function New-DbaAgentOperator {
+    <#
+    .SYNOPSIS
+        Creates a new operator on an instance.
+
+    .DESCRIPTION
+        If the operator already exists on the destination, it will not be created unless -Force is used.
+
+    .PARAMETER SqlInstance
+        The target SQL Server instance or instances. You must have sysadmin access and server version must be SQL Server version 2000 or greater.
+
+    .PARAMETER SqlCredential
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Operator
+        Name of the operator in SQL Agent.
+
+    .PARAMETER EmailAddress
+        The email address the SQL Agent will use to email alerts to the operator.
+
+    .PARAMETER NetSendAddress
+        The net send address the SQL Agent will use for the operator to net send alerts.
+
+    .PARAMETER PagerAddress
+        The pager email address the SQL Agent will use to send alerts to the oeprator.
+
+    .PARAMETER PagerDay
+        Defines what days the pager portion of the operator will be used. The default is 'Everyday'. Valid parameters
+        are 'EveryDay', 'Weekdays', 'Weekend', 'Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', and
+        'Saturday'.
+
+     .PARAMETER SaturdayStartTime
+        This a string that takes the Saturday Pager Start Time.
+
+    .PARAMETER SaturdayEndTime
+        This a string that takes the Saturday Pager End Time.
+
+    .PARAMETER SundayStartTime
+        This a string that takes the Sunday Pager Start Time.
+
+    .PARAMETER SundayEndTime
+        This a string that takes the Sunday Pager End Time.
+
+    .PARAMETER WeekdayStartTime
+        This a string that takes the Weekdays Pager Start Time.
+
+    .PARAMETER WeekdayEndTime
+        This a string that takes the Weekdays Pager End Time.
+
+    .PARAMETER IsFailSafeOperator
```

Comment:
Correct casing. `Failsafe` is used consistently so this would need to match if that is the perference.